Code/Framework/AzCore/AzCore/Memory/AllocatorBase.cpp

```diff
@@ -9,7 +9,8 @@
 #include <AzCore/Memory/Memory.h>
 #include <AzCore/Memory/AllocatorManager.h>
 
-using namespace AZ;
+namespace AZ
+{
```

Comment:
Missing indent for the namespace contents.

Reply:
umm didn't want to modify the entire file but that can be done

Reply:
I've reviewing with white space changes off and it's not too bad 👍